Riot/Categories/UITabBarController+Riot.m

```diff
@@ -0,0 +1,32 @@
+/*
+ Copyright 2017 Vector Creations Ltd
+ Copyright 2018 New Vector Ltd
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "UITabBarController+Riot.h"
+
+@implementation UITabBarController (Riot)
+
+- (UIViewController *)childViewControllerForStatusBarHidden
```

Comment:
We should use a `RiotTabBarController` instead.
Same remark as other categories:
> We should not override existing method in categories as said in Objective-C  [documentation](https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/ProgrammingWithObjectiveC/CustomizingExistingClasses/CustomizingExistingClasses.html#//apple_ref/doc/uid/TP40011210-CH6-SW4) this can lead to undefined behavior.

Reply:
Or put those methods directly in `MasterTabBarController` 🙂

Reply:
👍